skfp/preprocessing/filters/rule_of_2.py

```diff
@@ -0,0 +1,91 @@
+from typing import Optional
+
+from rdkit.Chem import Mol
+from rdkit.Chem.Crippen import MolLogP
+from rdkit.Chem.Descriptors import MolWt
+from rdkit.Chem.rdMolDescriptors import CalcNumHBA, CalcNumHBD
+
+from skfp.bases.base_filter import BaseFilter
+
+
+class RuleOf2(BaseFilter):
+    """
+    Rule of 2 defined  in [1].
+
+    Proposed rule for finding reagent to building block design.
+
+    The proposed reagents according to this formula should meet the following criteria:
+    - molecular weight <= 200 daltons
+    - HBA <= 4
+    - HBD <= 2
+    - logP <= 2
+
+    Parameters
+    ----------
+    allow_one_violation : bool, default=True
+        Whether to allow violating one of the rules for a molecule. This makes the
+        filter less restrictive, and is the part of the original definition of this
+        filter.
```

Comment:
Is this right, or should this be `False`?

Reply:
Should be False, error in docstring